commands/announce_events_happening_today.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Every morning, announce upcoming events for the day, and
+ * events that are due tomorrow.
+ *
+ * @author Yaw Boakye <wheresyaw@gmail.com>
+ */
+'use strict';
+
+const config = require('../config');
+const getEventsHappeningToday = require('../helpers').getEventsHappeningToday;
+const pluralize = require('../helpers').pluralize;
+
+module.exports = (channel, iconUrl, bot) => {
+  return () => {
+    getEventsHappeningToday().
+      then(events => {
+        if (events.length) {
+          let i, event,
+              attachments = [],
+              text = `${pluralize(events.length, 'event')} happening today:`;
+
+          for (i = 0; i < events.length; i++) {
+            event = events[i];
+
+            attachments.push({
+              fields: [
+                { title: 'Location', value: event.location },
+                { title: 'Start At', value: event.start, short: true },
+                { title: 'Ends At', value: event.end, short: true },
+                { title: 'Description', value: event.description }
+              ],
+
+              title: event.summary,
+              title_link: event.htmlLink,
+              author_name: event.creator.displayName,
+              author_url: `mailto:${event.creator.email}`,
+              color: config.ATTACHMENT_COLOR
+            });
+          }
+
+          bot.api.chat.postMessage({
```

Comment:
The `bot.reply` and `bot.say` functions take the same message object as `postMessage`, [Source](https://github.com/howdyai/botkit/blob/master/readme.md#single-message-replies-to-incoming-messages). I think we should stick to those for consistency:

`bot.say({
            text,
            channel,
            attachments,
            as_user: false,
            icon_emoji: ':calendar:',
            username: bot.identity.name
})`